src/sentry/static/sentry/app/views/eventsV2/eventDetails/transaction/eventBreakdown.tsx

```diff
@@ -157,11 +162,68 @@ class EventBreakdown extends React.Component<Props> {
       return null;
     }
 
-    // TODO: Dora to take over
-    // const results = this.generateCounts();
-
-    return null;
+    const results = this.generateStats();
+
+    return (
+      <StyledBreakdown>
+        <SectionHeading>{t('Ops Breakdown')}</SectionHeading>
+        {results.ops.map(currentOp => {
```

Comment:
Nit: You can shorten this to `currOp`. I like to use `prev`, `curr`, `next` to prefix variables because they are all 4 chars.

Reply:
Will do